umi_tools/group.py

```diff
@@ -403,7 +440,7 @@ def main(argv=None):
                         else:
                             # Add the 'UG' tag to the read
                             read.tags += [('UG', unique_id)]
-                            read.tags += [('FU', top_umi)]
+                            read.tags += [('BX', top_umi)]
```

Comment:
Yes... well, the UmiAwareMarkDuplicatesWithMateCigar command is still in alpha stage. Until the barcodes get somehow standardized... anything can happen.
BTW, allowing the user to set a specific tag and giving something as default tag should solve any future change in the stantards.